packages/auth/amplify_auth_cognito/example/integration_test/sign_in_sign_out_test.dart

```diff
@@ -83,6 +83,19 @@ void main() {
       );
     });
 
+    testWidgets('should be able to sign in after a failed attempt',
```

Comment:
Let's remove/update this since it doesn't actually test the issue

Reply:
ha, yeah good call - wrote this before actually resolving the bug.

Reply:
@dnys1 I added a corresponding test in the authenticator.